### PR TITLE
Developed a Copy of the New York Times Article Page

### PR DESCRIPTION
**Pull request by MarilenaRoque**
Added all the elements and style for the Page applicating all the theories learned about the position and floating elements. Including float, grid, box model, flexbox, relative position, different display types, CSS reset.  We also made the recommended validations with Linter and W3 Validator.

#NAVBAR
Position Sticky
Included Icons, Buttons, Logo, and Links
Defined background and border for the Navbar and its elements

#Main
Embedded images and video
Position the section and its elements
Style the text
Included Media Icons
Inflation Section with the Paragraphs overlapping the image

#Asides
Position of the Elements using float and flexbox
Embedded Images 
Included and style the images caption

#BottomNavigation

Positioned all the list with its links
Included the icons
Included the borders

#Footer
Styled and positioned the footer list

### CODE REVIEW

**Comment by rammazzoti2000** [comment]
### :raised_hands: Pull Request under the review by Alexandru Bangau :raised_hands:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,911 @@
...
+        <span class="material-icons">
+          search
+        </span>
+        <a href="#" class="bolder-titles space-cosmos">
+          SPACE & COSMOS</a
+        >
+      </div>
+
+      <div class="center-nav">
+        <img alt="New York Times Logo" src="assets/images/nyt-logo.png" />
```
Comment (rammazzoti2000) [inline index.html]: the image should be a link, wrap it in an anchor tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,911 @@
...
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- Aside Section -->
+
+    <div class="asides">
+         
+      <aside class="MostPopular">
```
Comment (rammazzoti2000) [inline index.html:612]: The More Popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) should be links.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,911 @@
...
+
+        <div class="article-item">
+          <img src="assets/images/tree.jpg" alt="" />
+          <p class="figure-caption-items">Namouni and Morais, NASA</p>
+          <p>A Nest of Alien Asteroids Orbits Our Sun</p>
+          <span class="smallcaption">April 28</span>
+        </div>
+      </aside>
+
+      <aside class="SpaceAstronomy rows-asside">
```
Comment (rammazzoti2000) [inline index.html:705]: In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

File: index.html [hunk 4]
```diff
@@ -0,0 +1,911 @@
...
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          3 Hospital Workers Gave Out Masks. Weeks Later, They All Were Dead.
+        </p>
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">More in Space and Astronomy</a></h5>
+      <aside class="SpaceAstronomy rows-asside">
```
Comment (rammazzoti2000) [inline index.html:669]: In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

File: index.html [hunk 5]
```diff
@@ -0,0 +1,911 @@
...
+          <p class="figure-caption-items">Kamioka Observatory</p>
+          <p>
+            Institute for Cosmic Ray Research, University of
+            Tokyo.Why the Big Bang Produced Something Rather Than Nothing</p>
+          <span class="smallcaption">April 27</span>
+        </div>
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">Editors’ Picks</a></h5>
+      <aside class="EditorPicks rows-asside">
```
Comment (rammazzoti2000) [inline index.html:746]: In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
#### Hello @MarilenaRoque and @krash46 :raised_hands: , my name is Aleaxndru and I am your code reviewer for this project, by the way awesome work :1st_place_medal: 
You did great work on this project, and have handled very well the use of flexbox and grid model.
- The readme file is descriptive, the pull request has meaningful title and description.
- The commits are short and descriptive.
- Good use of  HTML semantic :muscle: 
There are still a few issues that are preventing me from approving this project, please see the comments bellow
### :stop_sign: Required Changes :construction: :construction_worker_woman: 
- [x] The navbar logo should be a link to the [original page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0) :point_left: 
- [x] The More Popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) should be links. Wrap them in anchor tags with empty links (`href="#"`)
- [x] In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)); since the articles you have are different from the actual articles they have on the original page, use anchor tags with empty links (`href="#"`)
#### 
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### :stop_sign:  Status: Changes required.

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### :stop_sign: Status: Required Changes

**Comment by MarilenaRoque** [comment]
Thank you for the awesome review!

All the requirements for the approval were fitted. 

- The navbar is a link fo the original page
-The Most Popular Section is a link and also all the articles inside are links to articles on the original page.
-The Images and Subtitles in the More in Astronomy and Space are links for original articles on the original webpage

Additionally, we have added some new features, and style to the page looks more similar to the real one:

- Added functionality for the NavBar Buttons
-Added links for the Create Login Section
-Added links to the original page for the links inside the Text Container
-Set up the hover change color for the Subtitles of the Asides Articles

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :airplane: 

Hi @MarilenaRoque and @krash46, congratulations on fixing all the comments pointed out by the previous TSE :smile: On this note, I hereby approve your project :+1: However before merging to master, I have one optional comment I need you to address.
[OPTIONAL]
Kindly implement the sticky footer as you did for the nav

Good luck with your next project :+1: 
Do not forget to submit the project completion form after merging
_Happy coding_ :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**